### PR TITLE
MAINT: remove summit news from frontpage

### PR DESCRIPTION
**Pull request by bsipocz**
Summit has been held, remove announcement highlight from front page.

(I'm not sure if this works but will need CI for checking it as I cannot build the site locally)

### CODE REVIEW

**Comment by jarrodmillman** [comment]
I am not sure what we should do after a new item occurs and there is no new item. I would expect the last item to appear, but I don't have strong opinions.

**Comment by bsipocz** [comment]
The current rendering, with having just the link to the "News" doesn't look that awful. Falling back to the previous one is not really a solution for SP as that was the previous summit :)

**Comment by jarrodmillman** [comment]
Why not just change
```
Second Developer Summit will be held in Seattle – June 2-5, 2024
```
to
```
Second Developer Summit held in Seattle – June 2-5, 2024
```

File: content/news.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: News
-newsHeader: Second Developer Summit will be held in Seattle – June 2-5, 2024
-date: 2024-04-11
+newsHeader: Second Developer Summit was held in Seattle – June 2-5, 2024
```
Comment (stefanv) [inline content/news.md:3]:
```suggestion
newsHeader: Second Developer Summit held in Seattle – June 2-5, 2024
```

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**